common.yml

```diff
@@ -281,23 +281,23 @@ mongo_tools_variables:
       mongod_args: ""
       mongod_port: 33333
     ssl: &mongod_ssl_startup_args
-      mongod_args: "--sslMode requireSSL --sslCAFile testdata/ca-ia.pem --sslPEMKeyFile testdata/test-server.pem"
-      replsettest_ssl_config: "sslMode: \\\"requireSSL\\\",sslPEMKeyFile: \\\"testdata/test-server.pem\\\", sslCAFile: \\\"testdata/ca-ia.pem\\\", sslAllowInvalidHostnames: \\\"\\\""
+      mongod_args: "--sslMode requireSSL --sslCAFile common/db/testdata/ca-ia.pem --sslPEMKeyFile common/db/testdata/test-server.pem"
+      replsettest_ssl_config: "sslMode: \\\"requireSSL\\\",sslPEMKeyFile: \\\"common/db/testdata/test-server.pem\\\", sslCAFile: \\\"common/db/testdata/ca-ia.pem\\\", sslAllowInvalidHostnames: \\\"\\\""
```

Comment:
I was on the fence about where all the PEM files should live, but I think it's fine to leave them as-is in the `common` subpackage. Some testing code in `common` references them with a relative path and I felt it wasn't worth it to make changes there.

Reply:
That seems sensible to me! Does this mean that we can now delete the PEM files that had been moved into the `/testdata/` dir?

Reply:
Yep, although git is interpreting the deletion as a rename/move to the `common` directory.

If you ctrl+F for this string in the "Files changed" tab, both PEM files should show up:
```
.pem → common/db/testdata/
```

Reply:
Ah, yep; I see it now. Thanks!